samples/security/spire/README.md

```diff
@@ -0,0 +1,59 @@
+# Integrating SPIRE as a CA through Envoy's SDS API
+
+Further details about this integration can be found [here](https://istio.io/latest/docs/ops/integrations/spire).
+
+This sample deploys a setup of [SPIRE](https://github.com/spiffe/spire) (the SPIFFE Runtime Environment) as an example of [Envoy's SDS](https://www.envoyproxy.io/docs/envoy/latest/configuration/security/secret) API integration in Istio. For more information
+on SPIFFE and SPIRE specs, refer to the [SPIFFE Overview](https://spiffe.io/docs/latest/spiffe-about/overview/).
+
+Once SPIRE is deployed and integrated with Istio, we will use a modified version of the [sleep](/samples/sleep/README.md) service to validate that it's [Identity](https://spiffe.io/docs/latest/spiffe-about/spiffe-concepts/#spiffe-verifiable-identity-document-svid) did come from SPIRE. 
+
+## Usage
+
+1. Deploy SPIRE. This **must** be done prior to installing Istio in your cluster:
+
+    ```bash
+    $ kubectl apply -f spire-quickstart.yaml
+    ```
+
+2. Ensure that the deployment is completed until moving to the next step. 
+
+    This can be verified by waiting on the `spire-agent` pod to become ready:
+
+    ```bash
+    $ export SPIRE_POD=$(kubectl get pod -n spire -l app=spire-agent -o jsonpath="{.items[0].metadata.name}")
+    $ kubectl wait --for=condition=ready pod -n spire $SPIRE_POD
```

Comment:
can't we just do something like this?
```bash
kubectl wait pod --for=condition=ready -n spire -l app=spire-agent
```
(not sure if it works, but worth a try)

Reply:
nice one, it does work